private_sharing/models.py

```diff
@@ -348,6 +390,7 @@ class DataRequestProjectMember(models.Model):
     joined = models.BooleanField(default=False)
     authorized = models.BooleanField(default=False)
     revoked = models.BooleanField(default=False)
+    visible = models.BooleanField(default=True)
```

Comment:
Shouldn't this be on `DataRequestProjectMember`? (As it's specific to the member/project combination.)

Reply:
It is.